src/Service.Tests/GraphQLBuilder/MutationBuilderTests.cs

```diff
@@ -907,46 +907,6 @@ public void UpdateMutationWithNestedInnerObject_NullableListTypeNullableItems()
             Assert.IsFalse(inputObj.Fields[1].Type.InnerType().IsNonNullType(), "list fields should be nullable");
         }
 
-        [TestMethod]
```

Comment:
why remove this test?

Reply:
For testing MsSql nested schema generation correctly, we need to setup mock sqlmetadataprovider/authzResolver and other dependencies. (And also the FK directive). I have added thorough unit tests to `NestedMutationBuilderTests.cs` class.